go/runtime/host/protocol/types.go

```diff
@@ -234,10 +234,22 @@ type RuntimeCapabilityTEERakQuoteResponse struct {
 	Signature signature.RawSignature `json:"signature"`
 }
 
+// RPCKind is the RPC call kind.
```

Comment:
Should this rather be defined in `go/runtime/enclaverpc/api` and then used in all the places where `kind` is defined?